src/utils/get-start-script.ts

```diff
@@ -0,0 +1,9 @@
+export function getStartScript(scripts: Record<string, string> | undefined): 'dev' | 'start' | undefined {
+  if (scripts?.dev) {
```

Comment:
Nit: no need for `else` if the `if` before `return`s, which makes it a bit flatter.

```ts
  if (scripts?.dev) {
    return 'dev'
  } 
  if (scripts?.start) {
    return 'start'
  }
  ```

Reply:
I have updated this!